LMGRemoteData/Classes/Business/LMGRDBusinessRemoteSource.swift

```diff
@@ -38,7 +38,7 @@ public final class LMGRDBusinessRemoteSource: NSObject, LMGDABusinessRemoteSourc
     public func getForCollection(_ params: LMGDACollectionRequestParams, session: LMGDASession) throws -> LMGDAPagedResult<LMGDABusiness> {
```

Comment:
Does `LMGDACollectionRequestParams` already have a limit property...?

Reply:
Yes it does